app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -48,6 +49,7 @@ class KiwixMainActivity : CoreMainActivity() {
   override val navController by lazy { findNavController(R.id.nav_host_fragment) }
   override val drawerContainerLayout: DrawerLayout by lazy { navigation_container }
   override val drawerNavView: NavigationView by lazy { drawer_nav_view }
+  override val readerDrawerNavView: NavigationView by lazy { reader_drawer_nav_view }
```

Comment:
I was suggesting that `readerDrawerNavView` become `readerTableOfContentsDrawer`

>, this does not show the Reader Table of Contents Drawer, it unlocks it.